runners/org.osbuild.rhel90

```diff
@@ -0,0 +1,61 @@
+#!/usr/libexec/platform-python
```

Comment:
I think we should use `/usr/bin/python3`, all Python executables in `/usr/bin/` have it in their shebangs:

```
#!/usr/bin/python3
```

platform-python concept is gone in RHEL 9 and it's just a symlink to `/usr/bin/python3.9`.